gtsfm/evaluation/metrics_report.py

```diff
@@ -0,0 +1,198 @@
+import os
+from typing import Any, Dict, List, Tuple, Union
+
+import plotly.graph_objects as go
+import plotly.subplots as psubplot
+from tabulate import tabulate
+
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+SUBPLOTS_PER_ROW = 3
+SUMMARY_KEY = "summary"
+DATA_KEY = "full_data"
```

Comment:
can we import these from the other file, instead of re-defining them? I think we used `FULL_DATA_KEY` there?

Reply:
Thanks for updating this, resolved now